src/plugins/exitscript/exitscript.js

```diff
@@ -0,0 +1,144 @@
+/**
+* @title WET-BOEW Exit script plugin
+* @overview Plugin redirects users to non secure site
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @ipaksc
+*/
+( function( $, window, wb ) {
+"use strict";
+var componentName = "wb-exitscript",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	defaults = {},
+	i18nDict = {
+		en: {
+			"exitscript-warning": "Warning",
+			"exitscript-exitMsg": "You are about to leave a secure site, do you wish to continue?",
+			"exitscript-yesMsg": "Yes",
+			"exitscript-cancelMsg": "Cancel"
+		},
+		fr: {
+			"exitscript-warning": "Avertissement",
+			"exitscript-exitMsg": "Vous êtes sur le point de quitter un site sécurisé. Voulez-vous continuer?",
+			"exitscript-yesMsg": "Oui",
+			"exitscript-cancelMsg": "Annuler"
+		}
+	},
+
+/**
+* @method init
+* @param {jQuery Event} event Event that triggered the function call
+*/
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm,
+			settings;
+		if ( elm ) {
+			$elm = $( elm );
+			settings = $.extend(
+		true,
+		defaults, {},
+		window[ componentName ],
+		wb.getData( $elm, componentName )
+	);
+			i18nDict = i18nDict[ $( "html" ).attr( "lang" ) || "en" ];
+			i18nDict = {
+				warning: i18nDict[ "exitscript-warning" ],
+				exitMsg: i18nDict[ "exitscript-exitMsg" ],
+				yesMsg: i18nDict[ "exitscript-yesMsg" ],
+				cancelMsg: i18nDict[ "exitscript-cancelMsg" ]
+			};
+			$elm.trigger( "setModaldialog", settings );
+			var exitParam = "?" + encodeURIComponent( "targetUrl=" + String( this.href ) );
+
+			if ( settings.targetBlank ) {
+				exitParam = exitParam + encodeURIComponent( "&&targetBlank=true" );
+			}
+
+			// Set default propertie
+
+			if ( settings.displayModal !== false ) {
+				if ( !settings.targetBlank ) {
+					settings.targetBlank = "";
+				} else {
+					settings.targetBlank = "target='_blank'";
+				}
+				if ( !settings.exitMsg ) {
+					settings.exitMsg = i18nDict.exitMsg;
+				}
+				if ( !settings.yesMsg ) {
+					settings.yesMsg = i18nDict.yesMsg;
+				}
+				if ( !settings.cancelMsg ) {
+					settings.cancelMsg = i18nDict.cancelMsg;
+				}
+				var tpl = document.createElement( "div" );
+				tpl.id = "tm";
+				tpl.innerHTML = "<section id='sExitModal' class='mfp-hide modal-dialog modal-content overlay-def'>" +
```

Comment:
is the `id='sExitModal'` needed?

The prefer format for custom ID is to use `wb.getId()`